seed.js

```diff
@@ -1,116 +1,141 @@
 
 Debt has isolated us and made us feel alone and ashamed. We have come out of the shadows to fight back as individuals and as a collective. We are here because we are organizing to win debt relief and a better economic system for all.`,
-    },
-    medicalDebtCollective: {
-      group: 'medical-debt',
-      collective: 'Medical Debt Collective',
-      tag: 'For anyone who went into debt for health care.',
-      description: `### We are working together to plan actions, to develop debt resistance campaigns and to launch coordinated strikes.
+  },
+  medicalDebtCollective: {
+    group: 'medical-debt',
+    collective: 'Medical Debt Collective',
+    tag: 'For anyone who went into debt for health care.',
+    description: `### We are working together to plan actions, to develop debt resistance campaigns and to launch coordinated strikes.
 
 We fight because it is wrong that 40 percent of people in debt use credit cards to cover basic living costs including rent, food, and utilities. It is wrong that 62 percent of personal bankruptcies in the U.S. are linked to medical debt. It is wrong that students are leaving college owing an average of $35,000, and millions are in default. It is wrong that payday lenders earn high profits from poverty. And it is wrong that local court systems target poor people, disproportionately black and brown, and load them up with debt.
 
 We are different in many ways. Some of us are old, and some are young; we are from different parts of the country; we are diverse in race, ethnicity and religious background. A common belief unites us: everyone should have access to the goods and services they need to live without going broke or going into debt.
 
 Debt has isolated us and made us feel alone and ashamed. We have come out of the shadows to fight back as individuals and as a collective. We are here because we are organizing to win debt relief and a better economic system for all.`,
-    },
-    solidarityBloc: {
-      group: 'solidarity-bloc',
-      collective: 'Solidarity Bloc',
-      tag: 'For anyone organizing in solidarity with people in debt.',
-      description: `We organize in solidarity with those who are struggling under the weight of indebtedness for simply trying to access basic needs like healthcare, education and housing.
+  },
+  solidarityBloc: {
+    group: 'solidarity-bloc',
+    collective: 'Solidarity Bloc',
+    tag: 'For anyone organizing in solidarity with people in debt.',
+    description: `We organize in solidarity with those who are struggling under the weight of indebtedness for simply trying to access basic needs like healthcare, education and housing.
 
 We are committed to direct action, mutual aid and campaign support.`,
-    },
   },
 };
+
+const customUserFields = [
+  {
+    name: 'Debt Amount',
+    description: 'A number greater than 0',
+    field_type: 'text',
```

Comment:
Can you add a comment that this needs to be text for us to support the decimal?

Reply:
actually because discourse doesn't have that option, only 'dropdown', 'text', and 'confirm' (whatever that is)